spec/common/importers/keeperJsonImporter.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { Utils } from "jslib-common/misc/utils";
+
+import { KeeperJsonImporter as Importer } from "jslib-common/importers/keeperImporters/keeperJsonImporter";
+
+import { testData as TestData } from "./testData/keeperJson/testData";
+
+describe("Keeper Json Importer", () => {
+  const testDataJson = JSON.stringify(TestData);
+
+  it("should parse login data", async () => {
+    const importer = new Importer();
```

Comment:
These 3 lines of setup are used in each test. You should extract them to a `beforeEach` function: https://jasmine.github.io/api/3.9/global.html#beforeEach

Or you may have to use a (non-Jasmine) setup function that takes the orgId as an argument, to accommodate the last test.

Reply:
The testDataJson is only stringified once. I've moved the instantiation of the importer into a beforeEach but kept the call to `parse` in the tests as that is not technically setup but the Act of AAA. Also solves the issue with the OrgId on the last test.

Reply:
Personally I think it's okay to have that basic assertion in a common function, it's almost part of the setup before continuing with the 'real' test. But this does solve the orgId problem, so that's fine.